kore/src/main/scala/org/kframework/unparser/Unparse.scala

```diff
@@ -34,9 +36,13 @@ object Unparse extends {
 object ToKast {
   def apply(k: K): String = {
     val b = StringBuilder.newBuilder
-    unparse(b, false, 0, k)
+    unparse(s => b ++= s, false, 0, k)
     b.toString()
   }
+  def apply(k: K, out: PrintStream): Unit = {
```

Comment:
this was needed so that the entire string doesn't have to live in memory at once